### PR TITLE
Option to build kube-burner from source

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description

For dev purposes it's useful to provide the option of using a kube-burner binary built from source.

### CODE REVIEW

**Review by paigerube14 — APPROVED** [review]
/lgtm

File: workloads/kube-burner/common.sh [hunk 1]
```diff
@@ -54,7 +54,14 @@ collect_pprof() {
 
 run_workload() {
   local CMD
-  curl -sS -L ${KUBE_BURNER_URL} | tar -xzC /tmp/ kube-burner
+  if [[ -n ${BUILD_FROM_REPO} ]]; then
+    git clone --depth=1 ${BUILD_FROM_REPO} kube-burner
```
Comment (dry923) [inline workloads/kube-burner/common.sh:58]: do we want to make a tempdir for this first to make sure we dont collide w/ anything that might currently be in the directory?

Comment (rsevilla87) [reply workloads/kube-burner/common.sh]: fixed

File: workloads/kube-burner/common.sh [hunk 2]
```diff
@@ -54,17 +54,25 @@ collect_pprof() {
 
 run_workload() {
   local CMD
-  curl -sS -L ${KUBE_BURNER_URL} | tar -xzC /tmp/ kube-burner
+  local KUBE_BURNER_DIR 
+  KUBE_BURNER_DIR=$(mktemp -d)
+  if [[ -n ${BUILD_FROM_REPO} ]]; then
+    git clone --depth=1 ${BUILD_FROM_REPO} ${KUBE_BURNER_DIR}
+    make -C ${KUBE_BURNER_DIR} build
+    mv kube-burner/bin/amd64/kube-burner /tmp/kube-burner
```
Comment (dry923) [inline workloads/kube-burner/common.sh:62]: should be 
mv ${KUBE_BURNER_DIR}/bin/amd64/kube-burner /tmp/kube-burner

Comment (rsevilla87) [reply workloads/kube-burner/common.sh]: fixed

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**